buildSrc/src/main/java/Dependencies.kt

```diff
@@ -58,8 +57,8 @@ object Deps {
     const val kotlin_coroutines_android = "org.jetbrains.kotlinx:kotlinx-coroutines-android:${Versions.coroutines}"
 
     const val allopen = "org.jetbrains.kotlin:kotlin-allopen:${Versions.kotlin}"
-    const val osslicenses_plugin = "com.google.android.gms:oss-licenses-plugin:${Versions.osslicenses_plugin}"
-    const val osslicenses_library = "com.google.android.gms:play-services-oss-licenses:${Versions.osslicenses_library}"
+    const val aboutlibraries = "com.mikepenz:aboutlibraries:${Versions.aboutlibraries}"
+    const val aboutlibraries_plugin = "com.mikepenz.aboutlibraries.plugin:aboutlibraries-plugin:${Versions.aboutlibraries}"
```

Comment:
What's the solution then? Having a fork of the library managed by Mozilla? Manually building the library from a submodule? Something else?

As far as I'm concerned, I don't think the risk is high. The library is pinned and fetched from Maven Central which doesn't allow to overwrite artifacts. If you go that way, why trusting any other library at all? Even Google's blobs are not immune to malwares. How [robotelectric](https://github.com/robolectric/robolectric), [plexus-containers](https://github.com/codehaus-plexus/plexus-containers), [bcprov-jdk15on](https://www.bouncycastle.org/latest_releases.html) or [mockk](https://github.com/mockk/mockk) are any safer? For the most part, they are also managed by anonymous benevolents and may also be subject to malware injections. Yet they are already part of fenix.

Reply:
Another idea would be to still use Google's gradle plugin, but reimplement the library part in Fenix. After all, that's just a basic list view. I think I'll go for that if you still think the security risk is too high.

Reply:
You are right. We may not have been as strict with some other libraries in the past. For some we do not have much choice (Google's libraries), others are limited to testing code at least. Nevertheless we probably want to reduce that instead of adding to it. :)

I think having a custom view for the licenses to drop the library dependency sounds like an acceptable workaround to me (that rewrite probably could live in our Android Components eventually).

Reply:
Fair enough. I've updated my PR with the requested changes.